openshift_cli_installer/utils/cli_utils.py

```diff
@@ -421,3 +439,14 @@ def run_create_or_destroy_clusters(clusters, create, action, parallel):
             processed_clusters.append(result.result())
 
     return processed_clusters
+
+
+def assert_gcp_osd_user_input(clusters, gcp_service_account_file):
+    if any([_cluster["platform"] == GCP_OSD_STR for _cluster in clusters]):
+        if not gcp_service_account_file or not os.path.exists(gcp_service_account_file):
```

Comment:
No need to check if file exists, click do it for us
This function can be removed